sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5553,9 +5553,9 @@ def test_issue_7180():
 
 
 def test_pretty_Complement():
-    assert pretty(S.Reals - S.Naturals) == '(-oo, oo) \ Naturals()'
+    assert pretty(S.Reals - S.Naturals) == 'Interval(-oo, oo, True, True) \ S.Naturals'
```

Comment:
The use_unicode=False case should still do a pretty print, just not using any non-ASCII characters.